src/view-components/tabbed-view/tabbed-view.component.tsx

```diff
@@ -25,15 +12,24 @@ export default function TabbedView(props: any) {
 
   const [selected, setSelected] = React.useState(getInitialTab());
 
-  function getInitialTab() {
+  function getInitialTab(): number {
```

Comment:
We could remove the explicit return type - its already `number`.

Reply:
I think you forgot this one @donaldkibet - thanks 👍 !

Reply:
sure forgot this, fixed it.